dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -74,7 +74,7 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
                         }
                     });
                 }
-            } catch (GHException | HttpException e){
```

Comment:
You are sure we don't still also need `HttpException`? I guess `IOException` is a superset :thinking:

Reply:
Yes, it is!